### PR TITLE
feat(CLI): <- adds various new commands to that

### PR DESCRIPTION
**Pull request by gskapka**
...including:

```
cli.js signMessage <msg>
cli.js checkERC1820Exists
cli.js pushRawSignedTx <rawSignedTx>
cli.js getTransactionCount <ethAddress>
```

### CODE REVIEW

File: deleteme [hunk 1]
```diff
@@ -0,0 +1 @@
...b85760006107ba565b815b925050506106e8565b600160a060020a0390811660009081526020818152604080832086845290915290205416905092915050565b6000808061081d857f01ffc9a70000000000000000000000000000000000000000000000000000000061094c565b909250905081158061082d575080155b1561083d576000925050506106e8565b61084f85600160e060020a031961094c565b909250905081158061086057508015155b15610870576000925050506106e8565b61087a858561094c565b909250905060018214801561088f5750806001145b1561089f576001925050506106e8565b506000949350505050565b600160a060020a0382166000908152600260209081526040808320600160e060020a03198516845290915281205460ff1615156108f2576108eb83836107ef565b90506106e8565b50600160a060020a03808316600081815260208181526040808320600160e060020a0319871684529091529020549091161492915050565b7bffffffffffffffffffffffffffffffffffffffffffffffffffffffff161590565b6040517f01ffc9a7000000000000000000000000000000000000000000000000000000008082526004820183905260009182919060208160248189617530fa90519096909550935050505056fea165627a7a72305820377f4a2d4301ede9949f163f319021a6e9c687c292a5e2b2c4734c126b524e6c00291ba01820182018201820182018201820182018201820182018201820182018201820a01820182018201820182018201820182018201820182018201820182018201820
```
Comment (gitmp01) [inline deleteme:1]: Name suggest the file has to be deleted ;)

**Review by gitmp01 — CHANGES_REQUESTED** [review]
Looks good, few things though:
 - I think you mistakenly versioned a file, check comments
 - Also, maybe the `README.md` needs to be updated with the new command?
 - You forgot to bump minor :)

File: deleteme (hunk 1 above)
Comment (gskapka) [reply deleteme]: LOL, oops. Fixing!

**Review by gitmp01 — APPROVED** [review]
LGTM

**Comment by gskapka** [comment]
Thaaanks! Merging...

**Pull request merged**